### PR TITLE
panel meeting agenda filter spacing TM-3619

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do:
- [x] Filters should not show past search button, regardless of screen size

### CODE REVIEW

File: src/sass/_panelMeetingAgendas.scss [hunk 1]
```diff
@@ -59,6 +59,19 @@
...
+
+    @media screen and (min-width: 2390px) and (max-width: 2780px) {
+      padding-right: 10em;
+    }
+    
+    @media screen and (min-width: 2781px) and (max-width: 3188px) {
+      padding-right: 14em;
+    }
+    
+    @media screen and (min-width: 3188px) {
```
Comment (pveland621) [inline src/sass/_panelMeetingAgendas.scss:72]: The most nitpick thing of all time but does this min-width need to be 3189px?

Comment (SidelineCory24) [reply src/sass/_panelMeetingAgendas.scss:72]: it does lol

Comment (SidelineCory24) [reply src/sass/_panelMeetingAgendas.scss]: done

**Review by pveland621 — APPROVED** [review]
LGTM!

**Pull request merged**